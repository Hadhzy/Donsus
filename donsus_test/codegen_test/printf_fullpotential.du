# escape sequences
# literals

printf(2, "\n");
printf(2+4, "\n");
printf("hello", "\n");
printf(4.5, "\n");
printf((2*2)+1);

/*
2
6
hello
4.500000
5
*/