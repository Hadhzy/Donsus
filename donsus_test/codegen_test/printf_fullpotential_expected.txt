2
6
hello
4.500000
5